### DISCUSSION TITLE
RE: "For You (beta)"

### DISCUSSION THREAD

**Discussion by kindoflew** [community/community#13331] [Other Feature Feedback, Questions, & Ideas]
Can we move all `"{someone_I_follow} starred {some_repo}"` posts to the "For You" feed instead of the main feed? Those kinds of updates tend to push important events down below the fold. And since there currently isn't a "Hide Updates From This User" option, I think it'd be a nice alternative. Thanks!

**Comment by toastal** [comment]
`github.com##dashboard [aria-labelledby="feed-original"] .watch_started` is the current uBlock Origin rule to remove `{some follow} starred {some repository}` from the main Dashboard.

`github.com##aside[aria-label="Explore"]` is a good one too to remove the distracting Explore area from the homepage as well.

`github.com###dashboard .social-reactions` can remove all the distracting reaction emoji from the dashboard.

`github.com###dashboard [aria-labelledby="feed-original"] .follow` removes `{some follow} followed {some user}`.

To hide both the "followed" and "starred" in the original feed, you can combine them with: `github.com###dashboard [aria-labelledby="feed-original"] :is(.follow, .watch_started)`. This really helps clean up and separate the actual important feed from the social one.

Microsoft really wants to steal your attention and keep you on this social media Git forge.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
